### PR TITLE
chore: update doc about JSONSchema sync and remove useless codes

### LINKED ISSUE #1443: Update FAQ about sync plugin schema
https://github.com/apache/apisix-dashboard/blob/master/docs/FAQ.md#4-after-modifying-the-plugin-schema-or-creating-a-custom-plugin-in-apache-apisix-why-cant-i-find-it-on-the-dashboard

Do we need to update this question?

cc @nic-chen

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] Documentation
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
close #1443

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Why backend E2E will fail?

**Comment by juzhiyuan** [comment]
will fix #1485 ?

**Comment by nic-chen** [comment]
> will fix #1485 ?

yes, we don't use that script any more.

File: docs/FAQ.md [hunk 1]
```diff
@@ -33,27 +33,23 @@ If you are using Apache APISIX below v2.0, be aware that the data from the etcd
...
-If you have a custom plugin, make sure it is in the `apisix` directory and use the following command.
+Refer to https://github.com/apache/apisix/blob/master/doc/control-api.md#get-v1schema
 
-```sh
-$ api/build-tools/schema-sync.sh /path/to/apisix
-
-# e.g
-$ api/build-tools/schema-sync.sh /usr/local/apisix
-```
+3. Copy the exported schema.json to the conf directory in the Dashboard working directory
```
Comment (liuxiran) [inline docs/FAQ.md:50]: how about add a link to `conf directory` to tell user where it is?

Comment (nic-chen) [reply docs/FAQ.md:50]: sure, thanks!

**Comment by juzhiyuan** [comment]
@nic-chen You should request more reviewers

**Review by membphis — COMMENTED** [review]
LGTM, @spacewander do you have time to look at this PR? many thx

File: docs/FAQ.md [hunk 2]
```diff
@@ -33,27 +33,23 @@ If you are using Apache APISIX below v2.0, be aware that the data from the etcd
...
+Refer to the beginning in:
+https://github.com/apache/apisix/blob/master/doc/control-api.md
 
-2. Execute the following commands.
+2. Execute the following commands to export jsonchema on your APISIX server (if it is configured for non-local access, it does not need to be executed on your APISIX server, and the access IP and port should be modified accordingly)
 
 ```sh
-# `$version` is the version number of Apache APISIX, e.g. master or 2.1.
-$ api/build-tools/schema-sync.sh $version
+curl 127.0.0.1:9090/v1/schema> schema.json
```
Comment (starsz) [inline docs/FAQ.md:45]: How about

```
curl 127.0.0.1:9090/v1/schema > schema.json
```

File: docs/FAQ.md [hunk 3]
```diff
@@ -33,27 +33,23 @@ If you are using Apache APISIX below v2.0, be aware that the data from the etcd
...
-If you have a custom plugin, make sure it is in the `apisix` directory and use the following command.
+Refer to https://github.com/apache/apisix/blob/master/doc/control-api.md#get-v1schema
 
-```sh
-$ api/build-tools/schema-sync.sh /path/to/apisix
-
-# e.g
-$ api/build-tools/schema-sync.sh /usr/local/apisix
-```
+3. Copy the exported `schema.json` to the `conf` directory in the Dashboard working directory (About working please refer to https://github.com/apache/apisix-dashboard/blob/master/docs/deploy.md#working-directory)
```
Comment (starsz) [inline docs/FAQ.md:50]:
```
(About working please refer to https://github.com/apache/apisix-dashboard/blob/master/docs/deploy.md#working-directory)
```

Maybe

```
(About working directory, please ...)
```

Comment (nic-chen) [reply docs/FAQ.md]: updated, thanks.

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by membphis — APPROVED** [review]

**Pull request merged**